src/pages/player/Player.tsx

```diff
@@ -2,32 +2,93 @@ import { useParams } from "react-router";
 import PlayerStatsTable from "./components/PlayerStatsTable";
 import QuestRecomendations from "./components/QuestRecomendations";
 import { ScrollArea } from "@/components/ui/scroll-area";
+import { StarIcon } from "lucide-react";
+import { useEffect, useMemo, useState } from "react";
+import { useToast } from "@/hooks/use-toast";
 
 const Player = () => {
   const { username } = useParams();
+  const LOCAL_STORAGE_KEY = "RS3PD_v1_favorite_users";
+
+  const [favorites, setFavorites] = useState<string[]>([]);
+  const [foregroundColor, setForegroundColor] = useState("0, 0%, 100%");
+
+  useEffect(() => {
+    const data = localStorage.getItem(LOCAL_STORAGE_KEY);
+    if (data) {
+      setFavorites(JSON.parse(data));
+    }
+  }, []);
+
+  useEffect(() => {
```

Comment:
This seems to only work in dark mode when hosted locally and only in light mode when deployed to the web